Doc/library/contextlib.rst

```diff
@@ -185,6 +185,19 @@ Functions and classes provided:
           with cm as file:
               # Perform processing on the file
 
+   It can also be used as a stand-in for :ref:`asynchronous context managers <async-context-managers>`::
```

Comment:
The line max length is 79 characters, please reformat

Reply:
Fixed